packages/web-components/fast-components-msft/src/design-system-provider/index.ts

```diff
@@ -0,0 +1,650 @@
+import { attr, nullableNumberConverter, observable } from "@microsoft/fast-element";
+import {
+    DensityOffset,
+    DesignSystem,
+    DesignSystemDefaults,
+} from "@microsoft/fast-components-styles-msft";
+import {
+    designSystemProperty,
+    designSystemProvider,
+    DesignSystemProvider,
+    DesignSystemProviderTemplate as template,
+} from "@microsoft/fast-foundation";
+import { DesignSystemProviderStyles as styles } from "./design-system-provider.styles";
+
+const fromView: { mode: "fromView" } = {
+    mode: "fromView",
+};
+
+const fromViewNumber: { mode: "fromView"; converter: typeof nullableNumberConverter } = {
+    ...fromView,
+    converter: nullableNumberConverter,
+};
+
+@designSystemProvider({
+    name: "fast-design-system-provider",
+    template,
+    styles,
+})
+export class FASTDesignSystemProvider extends DesignSystemProvider
+    implements
+        Omit<
+            DesignSystem,
+            | "contrast"
+            | "direction"
+            | "fontWeight"
+            | "neutralForegroundDarkIndex"
+            | "neutralForegroundLightIndex"
+        > {
+    /**
+     * Define design system property attributes
+     */
+    @attr({ attribute: "background-color", ...fromView })
+    @designSystemProperty({
+        cssCustomProperty: "background-color",
+        default: DesignSystemDefaults.backgroundColor,
+    })
+    public backgroundColor: string;
+
+    @attr({ attribute: "accent-base-color", ...fromView })
```

Comment:
No need for the `@attr` or `@observable` anymore - see the `fast-components` version